openstack/client.go

```diff
@@ -113,10 +113,15 @@ func Authenticate(client *gophercloud.ProviderClient, options gophercloud.AuthOp
 
 // AuthenticateV2 explicitly authenticates against the identity v2 endpoint.
 func AuthenticateV2(client *gophercloud.ProviderClient, options gophercloud.AuthOptions, eo gophercloud.EndpointOpts) error {
+	return v2auth(client, "", &options, eo)
+}
+
+// AuthenticateV2Ext explicitly authenticates against the identity v2 endpoint.
+func AuthenticateV2Ext(client *gophercloud.ProviderClient, options tokens2.AuthOptionsBuilder, eo gophercloud.EndpointOpts) error {
```

Comment:
It stood for Extended. Just trying to come up with a different name to not break the API of the existing function.

Reply:
I dropped it since you guys flagged this as being a major version bump and incorporated it into the regular V2 call. Since you're reading gopher cloud 2.x I figured it would be okay.